### DISCUSSION TITLE
Add filter for HL7 and add support for DICOM protocol

### DISCUSSION THREAD

**Discussion by nathan-malinowski** [GyulyVGC/sniffnet#338] [Ideas]
It would be really nice to have if sniffnet could implement HL7/FHIR filter and the DICOM protocol

https://www.wireshark.org/docs/dfref/h/hl7.html
https://wiki.wireshark.org/Protocols/dicom.md

more info
https://hl7-definition.caristix.com/v2/
https://dicom.nema.org/medical/dicom/current/output/html/part07.html
https://dicom.nema.org/medical/dicom/current/output/html/part08.html
https://dicom.nema.org/medical/dicom/current/output/html/part18.html

**Comment by GyulyVGC** [comment]
Hi @nathan-malinowski sorry for the delay in the reply.

I've implemented a possible fix to this request in https://github.com/GyulyVGC/sniffnet/pull/414.
Now Sniffnet supports filtering custom values/ranges of ports: thanks to this filter you should be able to filter HL7 (port 2575), DICOM (port 104), and all the services you desire.

These features are merged into the main branch and will be officially released in v 1.3.

**Reply by nathan-malinowski** [reply]
Awesome, thank you!

**Comment by GyulyVGC** [comment]
Additionally, traffic detection for more than 6000 services, including HL7 and DICOM, has been implemented in #450 
@nathan-malinowski
